tests/console/suseconnect_scc.pm

```diff
@@ -30,8 +30,9 @@ sub run {
 
     select_serial_terminal;
     die 'SUSEConnect package is not pre-installed!' if script_run 'command -v SUSEConnect';
-    if ((is_jeos || is_sle_micro) && script_run(qq(SUSEConnect --status-text | grep -i 'not registered'))) {
-        die 'System has been already registered!';
+    if ((is_jeos || is_sle_micro)) {
+        my $status = script_output('SUSEConnect --status-text');
+        die 'System has been already registered!' if ($status !~ m/not registered/i);
```

Comment:
But in case of failure, like in https://openqa.suse.de/tests/15456552#step/suseconnect_scc/13  the error message reported by openQA will not be the one in the "die", but `Error: zypper returned -1 with '<?xml version='1.0'?>` as an example.